### PR TITLE
Rolling to V8 10.4.132.5

### PR DESCRIPTION
**Pull request by denobot**

### CODE REVIEW

File: src/isolate.rs [hunk 1]
```diff
@@ -151,8 +151,14 @@ pub type NearHeapLimitCallback = extern "C" fn(
...
 ) -> usize;
 
-pub type LegacyOomErrorCallback =
-  extern "C" fn(location: *const c_char, is_heap_oom: bool);
+#[repr(C)]
+pub struct OomDetails {
+  pub is_heap_oom: bool,
+  pub detail: *const c_char,
+}
+
```
Comment (littledivy) [inline src/isolate.rs:159]: https://github.com/denoland/v8/blob/a5ae0382548204c60025cb26618231ba48ad83f1/include/v8-callbacks.h#L223-L226

```suggestion

impl Default for OomDetails {
  fn default() -> Self {
    Self {
      is_heap_oom: false,
      detail: ptr::null(),
    }
  }
}
```

**Review by littledivy — APPROVED** [review]
LGTM

File: src/isolate.rs (hunk 1 above)
Comment (ry) [reply src/isolate.rs:159]: I don't think its ever constructed by the user - only by V8... so no point in having a default implementation.

**Review by ry — APPROVED** [review]
LGTM

**Comment by ry** [comment]
Ideally these `OomDetails` changes would be landed in a separate PR from the upgrade.

**Review by AaronO — APPROVED** [review]

**Pull request merged**